File: JHY/C_Algorithm/C_Algorithm/lv9/1978.c
#include <stdio.h>
#include <stdbool.h>

bool prime(int n);

int main(void)
{
    int n, m, number=0;
    scanf("%d", &m);
    for (int i = 0; i < m; i++) {
        scanf("%d",&n);
        if(prime(n) == true)
            number++;
    }
    printf("%d\n", number);
    return 0;
}

bool prime(n)
{
    int i = 0;
    for(i = 2; i < n; i++)
    {
        if(n % i == 0)
            break;
    }
    if(n == i)
        return true;
    else
        return false;
}

File: LSW/aug/3/1978.py
#! /opt/homebrew/bin/python3.9
def prime(number):
    if number != 1:
        for f in range(2, number):
            if number % f == 0:
                return False
    else:
        return False
    return True


v = int(input())

number = list(map(int, input().split()))
count = 0
for i in number:
    if prime(i):
        count += 1
print(count)
